Client/Frontend/Browser/QRCode.xcassets/Contents.json

```diff
@@ -1,6 +1,6 @@
 {
   "info" : {
-    "version" : 1,
-    "author" : "xcode"
+    "author" : "xcode",
+    "version" : 1
   }
-}
\ No newline at end of file
+}
```

Comment:
Harmless change - I guess this happened on opening the file?

Reply:
yeah. lol